lib/algora_web/components/core_components.ex

```diff
@@ -465,7 +465,7 @@ defmodule AlgoraWeb.CoreComponents do
       id={@id}
       phx-mounted={@show && show_modal(@id)}
       phx-remove={hide_modal(@id)}
-      class="relative z-50 hidden"
+      class="relative z-[1001] hidden"
```

Comment:
Increased the z-index of the modal since the chat panel has a wild z-index (1000)